OTAnalytics/plugin_prototypes/track_visualization/track_viz.py

```diff
@@ -57,7 +57,7 @@
 from OTAnalytics.plugin_filter.dataframe_filter import DataFrameFilterBuilder
 
 """Frames start with 1 in OTVision but frames of videos are loaded zero based."""
-FRAME_OFFSET = 1
+FRAME_OFFSET = 0
```

Comment:
Has this changed?

Reply:
Yes, because we use the frame information to filter the tracks and to find the matching background image. This offset is a relict of the old strategy.